### DISCUSSION TITLE
Reduce cpu usage (as in stop rendering loop) when no user action ?

### DISCUSSION THREAD

**Discussion by ThomasCartier** [bevyengine/bevy#14557] [Q&A] [answered]
Hi,

Just a quick question: is there a way to reduce the cpu usage dynamically by stopping the rendering loop ?
The use case is a menu that would stop the rendering loop when user doesn't do any action during 10 seconds for example. 
That would lower the cpu usage / increase battery life when idling.

Any hint ?

Thanks

**Comment by JMS55** [comment, marked as answer]
Look at https://dev-docs.bevyengine.org/bevy/winit/struct.WinitSettings.html

**Reply by rparrett** [reply]
We have a helpful example for this as well: https://bevyengine.org/examples/window/low-power/

**Reply by ThomasCartier** [reply]
Thanks a lot guys,
exactly what I needed (sorry for missing this example, didn't scroll up to this category)
